go.mod

```diff
@@ -1,22 +1,21 @@
 module github.com/siderolabs/discovery-client
 
-go 1.22.1
+go 1.22.2
 
 require (
-	github.com/cenkalti/backoff/v4 v4.2.1
+	github.com/cenkalti/backoff/v4 v4.3.0
 	github.com/siderolabs/discovery-api v0.1.4
 	github.com/siderolabs/gen v0.4.8
 	go.uber.org/zap v1.27.0
-	google.golang.org/grpc v1.62.1
+	google.golang.org/grpc v1.63.2
```

Comment:
please don't update to 1.63 yet, it has an issue which leads to segfaults in Talos